include/mega/megaclient.h

```diff
@@ -77,15 +77,12 @@ class MEGA_API MegaClient
     void confirmsignuplink(const byte*, unsigned, uint64_t);
     void setkeypair();
 
-    /**
-     * @brief Initialises the Ed25519 EdDSA key user properties.
-     *
-     * A key pair will be added, if not present, yet.
-     *
-     * @return Error code (default: 1 on success).
-     */
+    // initialises the Ed25519 EdDSA key user properties
```

Comment:
This is the function signature, why has the Doxygen string been removed here?

It (A) should have a docstring, and (B) parameters/return values should be documented.